S3/s3_get_file.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+
+import sys
+import boto3
+from optparse import OptionParser
+ACCESS_KEY = "AKIAI3OWYQBREX3NSAHQ"
```

Comment:
Yes I created new keys and deactivated the old ones, it should be setup in the environment